plugins/anchor-links/index.js

```diff
@@ -110,6 +145,7 @@ function processHeading(node, compatibilitySlug, links, headings) {
     permalinkSlug,
     slug,
     title,
+    tabbedSectionDepth,
```

Comment:
Each item pushed to the `headings` array will now have a `tabbedSectionDepth` property. Headings that are _not_ nested within `<Tabs />` will have `tabbedSectionDepth === 0`. In `dev-portal`, we'll avoid rendering such headings in our table of contents.